docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -30,6 +30,21 @@ This page explains how to get started monitoring the security posture of your cl
 
 You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access.
 
+[discrete]
+[[cspm-aws-agentless]]
+== Agentless deployment option
+beta::[]
+
+The steps to create an agentless deployment are similar to those to create an agent-based deployment. To deploy using agentless technology, follow the instructions below with the following modifications:
```

Comment:
This is not correct statement, as agentless deployment is simplifying the integration setup. And Cloudformation option in case of agentless is about creation of a role only and in case of agent-based it is creation of the ec2 instance with agent installed on it. For simplicity I would propose to keep the header for agentless and agent based deployment options and detail out the steps in each of these sections.